internal/secrets/gcpkms/kms_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gcpkms
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/go-cloud/internal/testing/setup"
+	"google.golang.org/api/option"
+
+	cloudkms "cloud.google.com/go/kms/apiv1"
+	"github.com/google/go-cloud/internal/secrets/driver"
+	"github.com/google/go-cloud/internal/secrets/drivertest"
+)
+
+const (
+	// These constants capture values that were used during the last --record.
```

Comment:
Move the doc comment to right before the `const` keyword. Otherwise, the doc is just associated with `projectID`.